docs/docs/concepts/jobs.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Jobs
+sidebar_position: 6
+---
+
+Jobs can be handled with [dm-job](https://github.com/equinor/dm-job).
+
+This is an API where you can:
+- Start jobs
+- Stop jobs
+- Request the status for a running job
+- Get result from a job
+
+
+If your custom application want to use dm-job, you have to create a JobHandler file for your job.
+
+This JobHandler must have the name `__init__.py` and be placed inside a folder called `job_handler_plugins`.
```

Comment:
Why this folder? Babak seems to have placed his somewhere else. And is it important where I put the folder?

Reply:
ah, the docs I wrote here are not entirely correct.

The job handler files must be available inside the dm-job container, and the dm-job API will look for job handler files inside the folder dm-job/src/default_job_handlers.

The way we have done this with dm-core-packages is to create a docker volume, which means that we "copy" the content of "dm-core-packages/example/job_handlers" into the folder "dm-job/src/job_handler_plugins"

will update this